pkg/storage/export/export/export.go

```diff
@@ -346,14 +351,20 @@ func (ctrl *VMExportController) Init() error {
 	if err != nil {
 		return err
 	}
-	ctrl.instancetypeMethods = &instancetype.InstancetypeMethods{
-		InstancetypeStore:        ctrl.InstancetypeInformer.GetStore(),
-		ClusterInstancetypeStore: ctrl.ClusterInstancetypeInformer.GetStore(),
-		PreferenceStore:          ctrl.PreferenceInformer.GetStore(),
-		ClusterPreferenceStore:   ctrl.ClusterPreferenceInformer.GetStore(),
-		ControllerRevisionStore:  ctrl.ControllerRevisionInformer.GetStore(),
-		Clientset:                ctrl.Client,
-	}
+	ctrl.instancetypeHandler = instancetypeapply.NewVMApplier(
+		find.NewSpecFinder(
```

Comment:
really nit: change import find to instanccetypeFind to make it similar to the preferenceFind import call, to get some context on where this find comes from